internal/backend/sessions/sessionworkflows/history.go

```diff
@@ -70,6 +74,10 @@ func (ws *workflows) GetWorkflowLog(ctx context.Context, filter sdkservices.Sess
 			}
 
 			riter.rs = append(riter.rs, event)
+
+			// On a descending query, we need to prepopulate the events so they'll be available when
```

Comment:
I can guess that you meant "query in descending order". But bump where? Prepopulate where?

I'm looking at the raw code in this function and I don't understand what this comment is trying to explain, i.e. how `events[event.EventId] = event` relates to this text.